main/acle.md

```diff
@@ -9382,6 +9415,2348 @@ are named after. All of the functions have external linkage.
+  // _za64[_s64], _za64[_u64] and _za64_[f64]
+  __attribute__((arm_streaming, arm_shared_za))
+  void svwrite_ver_za8[_s8]_vg4(uint64_t tile, uint32_t slice_base,
+                                uint64_t slice_x4_offset, svint8x4_t zn);
+  
+  
+  // Variants are also available for _za64[_u64] and _za64[_f64]
+  __attribute__((arm_streaming, arm_shared_za))
+  void svwrite_za64[_s64]_vg1x2(uint32_t slice_base, uint64_t slice_offset,
+                                svint64x2_t zn);
+  
+  
+  // Variants are also available for _za64[_u64] and _za64[_f64]
+  __attribute__((arm_streaming, arm_shared_za))
+  void svwrite_za64[_s64]_vg1x4(uint32_t slice_base, uint64_t slice_offset,
+                                svint64x4_t zn);
+  
+  ```
+
+#### PTRUE
+
+Initialise predicate-as-counter to all active
+
+``` c
+  // Variants are also available for _c16, _c32 and _c64
+  __attribute__((arm_streaming))
+  svcount_t svptrue_c8();
+  
+  ```
+
+#### PEXT
+
+Transform a predicate-as-counter to a predicate (pair).
+
+``` c
+  // Variants are also available for _c16, _c32 and _c64
+  __attribute__((arm_streaming))
+  svbool_t svpext_lane_c8(svcount_t pnn, uint64_t imm);
+  
+  
+  // Variants are also available for _c16, _c32 and _c64
+  __attribute__((arm_streaming))
+  svboolx2_t svpext_lane_c8_x2(svcount_t pnn, uint64_t imm);
+  
+  ```
+
+#### PSEL
+
+Predicate select between predicate value or all-false
+
+``` c
+  // Variants are also available for _b16, _b32 and _b64
+  __attribute__((arm_streaming_compatible))
+  svbool_t svpsel_lane_b8(svbool_t pn, svbool_t pm, uint32_t idx, uint64_t imm);
+  
+  ```
+
+#### CNTP
+
+Set scalar to count from predicate-as-counter. ``vl`` is expected to be 2 or 4.
+
+``` c
+  // Variants are also available for _c16, _c32 and _c64
+  __attribute__((arm_streaming))
+  uint64_t svcntp_c8(svcount_t pnn, uint64_t vl);
+  
+  ```
+
+#### UCLAMP, SCLAMP, FCLAMP
+
+Multi-vector clamp to minimum/maximum vector
+
+``` c
+  // Variants are also available for _s8, _u8, _s16, _u16, _s32, _u32, _f32,
+  // _s64, _u64 and _f64
+  __attribute__((arm_streaming_compatible))
+  svfloat16_t svclamp[_f16](svfloat16_t zd, svfloat16_t zn, svfloat16_t zm);
+  
+  
+  // Variants are also available for _single_s8_x2, _single_u8_x2,
+  // _single_s16_x2, _single_u16_x2, _single_s32_x2, _single_u32_x2,
+  // _single_f32_x2, _single_s64_x2, _single_u64_x2 and _single_f64_x2
+  __attribute__((arm_streaming))
+  svfloat16x2_t svclamp[_single_f16_x2](svfloat16x2_t zd, svfloat16_t zn,
+                                        svfloat16_t zm);
+  
+  
+  // Variants are also available for _single_s8_x4, _single_u8_x4,
+  // _single_s16_x4, _single_u16_x4, _single_s32_x4, _single_u32_x4,
+  // _single_f32_x4, _single_s64_x4, _single_u64_x4 and _single_f64_x4
+  __attribute__((arm_streaming))
+  svfloat16x4_t svclamp[_single_f16_x4](svfloat16x4_t zd, svfloat16_t zn,
+                                        svfloat16_t zm);
+  
+  ```
+
+#### REVD
```

Comment:
I believe this instructions are only SME or SVE2.1. 
https://developer.arm.com/documentation/ddi0602/2022-09/SVE-Instructions/REVD--Reverse-64-bit-doublewords-in-elements--predicated--?lang=en

Reply:
Well spotted, I'll remove them.

Reply:
Addressed in https://github.com/ARM-software/acle/pull/217/commits/18c5da294a88c3a4fbb4c8d3301c5f71293f1aff